Algo.cpp

```diff
@@ -5,4 +5,8 @@ Algorithm::Algorithm(tsp::Graph<double, std::vector> &graph_)
   std::cout << "New algorithm constructed" << std::endl;
 }
 
-Algorithm::~Algorithm() { std::cout << "Algorithm deconstructed" << std::endl; }
\ No newline at end of file
+Algorithm::~Algorithm() { std::cout << "Algorithm deconstructed" << std::endl; }
+
+const std::vector<unsigned> &Algorithm::getSolution() const {

```

Comment:
in project we do not use CamelCase but snake_case in functions/methods. CamelCase only in ClassNames

Reply:
+1